main/stm32bl.c

```diff
@@ -86,11 +114,46 @@ int stm32bl_go(uint32_t addr)
     if (wait_resp() != STM32BL_ACK) {
         return ESP_FAIL;
     }
-
     send_address(addr);
     return wait_resp();
 }
 
+static int stm32bl_run_raw_code(uint32_t target_address, const uint8_t *code, uint32_t code_size)
+{
+	uint32_t stack_le = 0x20002000;
```

Comment:
What does `_le` stand for? And is it a constant? If yes, I suggest to define it in the header.